### ISSUE TITLE
Assigning Shader to specified GPU on a workstation with two GPUs

### ISSUE THREAD

**Issue by DetermLZ** [godotengine/godot#78702]
### Godot version

3.5.1-stable

### System information

Win 10, AMD Threadripper PRO 5995WX, 2 x RTX A5000

### Issue description

Hello!

I am using Godot 3.5.1 to render point clouds in real-time. My approach is to render each point cloud with a specified MeshIntance by uploading data to a corresponding Shader with GDNative C++. So each point cloud has a paired MeshInstance and a Shader. 

My workstation has 2 Nvidia TRX A5000 GPUs connected with NV Link, but Godot only uses the first GPU, and the second GPU is not used at all. All the rendering tasks are allocated to the first GPU. The workflow looks like this:
![image](https://github.com/godotengine/godot/assets/111273475/3363861d-8c18-4438-8c98-0d72eb1c766f)

I know it is feasible to specify a dedicated GPU for rendering when a desktop has an integrated GPU and a dedicated GPU. 
But is it possible to assign the tasks of each Shader to a specified GPU and use all GPUs simultaneously? Does Godot 3.5.1 support this feature? If not, is there a hack way to do this? 

The desired result looks like this:
![image](https://github.com/godotengine/godot/assets/111273475/a81c9be3-2c05-476d-a2b0-57c85b4a319d)

Any help will be appreciated.

### Steps to reproduce

Not applicable.

### Minimal reproduction project

Not applicable.

**Comment by DetermLZ** [comment]
By the way, I would also to know if it is possible the specify the GPU in Godot 4?

**Comment by Calinou** [comment]
> By the way, I would also to know if it is possible the specify the GPU in Godot 4?

Yes, using the `--gpu-index` command line argument. Godot prefers the first dedicated GPU by default.

**Comment by AThousandShips** [comment]
For support questions and help please turn to the other [community channels](https://godotengine.org/community), this is for bug reporting, not a help desk.
